packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -23,16 +23,35 @@ export class TreeView extends FASTElement {
     @observable
     private lastFocused: HTMLElement;
 
+    @observable
+    private nested: boolean;
+
     @observable slottedTreeItems: HTMLElement[];
     private slottedTreeItemsChanged(oldValue, newValue): void {
         if (this.$fastController.isConnected) {
             // filter the tree items until that's done for us in the framework
             this.resetItems();
             this.treeItems = this.getVisibleNodes();
             this.setItems();
+
+            // check if any tree items have nested items
+            // if they do, apply the nested attribute
+            if (this.checkForNestedItems()) {
+                this.slottedTreeItems.forEach(node => {
+                    if (isTreeItemElement(node)) {
+                        node.setAttribute("nested", "true");
```

Comment:
assuming `nested` is a boolean attribute (see above comment) should this just set empty string?

Reply:
yeah, likely